lib/modules/filteReddit.js

```diff
@@ -746,6 +755,17 @@ module.go = function() {
 			toggleNsfwFilter(toggle, true);
 		}
 	);
+
+	if (isPageType('linklist')) {
+		if (module.options.showRemoveVisitedTab.value && process.env.BUILD_TARGET !== 'safari') {
+			CreateElement.tabMenuItem({
```

Comment:
I don't think it's well placed there, as one would then have to scroll before initiating the option. 

Perhaps as an alternative could be to place a `filter by` line at top of the linklist, similar to 'navigate by'.
![image](https://cloud.githubusercontent.com/assets/1748521/15631094/a12c412e-2556-11e6-9afc-48af52a0d972.png)